doc/howto_release.md

```diff
@@ -185,17 +175,22 @@ good results for the first release candidate because it contains contributor han
 and can identify new contributors, so use with the _api_ backend, e.g.:
 
 ```bash
-python ./utils/generate_release_notes.py api releasebranch_8_4 8.3.0 $VERSION
+python ./utils/generate_release_notes.py api $(git rev-parse --abbrev-ref HEAD) 8.3.0 $VERSION
```

Comment:
We have to remember that it is probably not always relative to 8.3.0. The previous line made it clear to me when reading, the new line not as much